src/prefect/orion/services/flow_run_notifications.py

```diff
@@ -142,7 +142,7 @@ def get_ui_url_for_flow_run_id(self, flow_run_id: UUID) -> str:
             flow_run_id: the flow run id.
         """
         api_url = PREFECT_API_URL.value() or "http://ephemeral-orion/api"
-        ui_url = api_url.strip("/api")
+        ui_url = api_url.replace("/api", "")
```

Comment:
This isn't a super robust fix since we technically want to replace the rightmost `/api` once I think. I'm planning on more serious fixes for this later though. Thanks for contributing!

Reply:
Ah I think this actually won't work since the Cloud URL is `api.prefect.io/api`